Formula/ipython.rb

```diff
@@ -21,6 +21,11 @@ class Ipython < Formula
   depends_on "six"
   depends_on "zeromq"
 
+  resource "appnope" do
```

Comment:
this is still needed (but just on mac): https://github.com/ipython/ipython/blob/311a3a0fd25d09cc4d314f900e31ee6ac2cfbba4/setup.cfg#L32